docker/Dockerfile.ci_gpu

```diff
@@ -85,6 +85,9 @@ RUN bash /install/ubuntu_install_tflite.sh
 COPY install/ubuntu_install_caffe2.sh /install/ubuntu_install_caffe2.sh
 RUN bash /install/ubuntu_install_caffe2.sh
 
+COPY install/ubuntu_install_dgl.sh /install/ubuntu_install_dgl.sh
+RUN bash /install/ubuntu_install_dgl.sh
+
 RUN pip3 install Pillow
```

Comment:
Why not just add dgl here?

Reply:
I am just trying to follow the convention of installing external frameworks.
Please see docker/install/ubuntu_install.mxnet.sh